### PR TITLE
fix: add Wasiq Amjad Bhamla

### PR DESCRIPTION
**Pull request by WasiqB**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/892"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
1. Could you please revert the `package-lock.json` file!

2. Change the PR title to - `fix: add Wasiq Amjad Bhamla`

**Comment by WasiqB** [comment]
@syedsohan , changes made as per review comments.

**Review by syedsohan — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

File: public/data/wasiqb.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "color": "blue",
+      "description": "My first contribution to other project",
+      "url": "https://github.com/AppiumTestDistribution/AppiumTestDistribution"
+    },
+    {
+      "title": "Silver winner in Applitools UltraFast Cross Browser Testing Hackathon",
+      "date": "July 2020",
+      "icon": "dollar",
+      "color": "red",
+      "description": "Won $200 in the Hackathon organized by Applitools for UltraFast Cross browser testing. Was placed in Top 100 out of 2000+ applications.",
```
Comment (krishguptadev) [inline public/data/wasiqb.json:62]:
```suggestion
      "description": "Won $200 in the Hackathon organized by Applitools for UltraFast Cross-browser testing. Was placed in Top 100 out of 2000+ applications.",
```

File: public/data/wasiqb.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      "color": "green",
+      "description": "Open sourced my 2nd project for Web automation testing",
+      "url": "https://github.com/WasiqB/coteafs-selenium"
+    },
+    {
+      "title": "AppiumTestDistribution",
+      "date": "May 2019",
+      "icon": "github",
+      "color": "blue",
+      "description": "My first contribution to other project",
```
Comment (krishguptadev) [inline public/data/wasiqb.json:54]:
```suggestion
      "description": "My first contribution to a project",
```

File: public/data/wasiqb.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      "color": "black",
+      "description": "My first serious open source project for Mobile Automation testing",
+      "url": "https://github.com/WasiqB/coteafs-appium"
+    },
+    {
+      "title": "coteafs-selenium",
+      "date": "April 2018",
+      "icon": "github",
+      "color": "green",
+      "description": "Open sourced my 2nd project for Web automation testing",
```
Comment (krishguptadev) [inline public/data/wasiqb.json:46]:
```suggestion
      "description": "Open-sourced my 2nd project for Web automation testing",
```

File: public/data/wasiqb.json (hunk 2 above)
Comment (WasiqB) [reply public/data/wasiqb.json:54]: @krishguptadev , This was my first contribution to other peoples open source project. I had already contributed to my own projects. Hence, mentioned `other project`.

Comment (krishguptadev) [reply public/data/wasiqb.json:54]: It would be ... ig

```suggestion
      "description": "My first contribution to another project",
```

**Pull request merged**